### DISCUSSION TITLE
HTTP Exception 500: "CC-BY" is not a valid backing value for enum App\Enum\LicenseType (View: /var/www/html/Lychee/resources/views/livewire/pages/gallery/album.blade.php)

### DISCUSSION THREAD

**Discussion by mniemela** [LycheeOrg/Lychee#2391] [Bugs] [answered]
### Lychee version

v5.2.0

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

I just upgraded from Lychee 4.8.1 through 4.13.0 to 5.2.0 and I get this error when trying to access old folders.

![image](https://github.com/LycheeOrg/Lychee/assets/7679721/253a0a52-2099-412a-9ad9-c7f4baf02776)

### Steps to reproduce the issue

1. Upgrade from Lychee 4.8.1 through 4.13.0 to 5.2.0 with some old names for licences in some folders
2. After migration, go to an old folder with old licence name. Error should be visible.

EDIT: In Lychee 4.13.0 you get "Internal server error" -banner instead of error listed here.
EDIT2: Lychee 4.12.0 works fine, so I guess this is introduced by the change to enum-based licence?

### Diagnostics [REQUIRED]

No errors visible in diagnostics.

### Browser & System [REQUIRED]

Edge & Windows 11

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment, marked as answer]
Fixed with https://github.com/LycheeOrg/Lychee/pull/2393
